docs/tryout.md

```diff
@@ -0,0 +1,390 @@
+        "token_type":"Bearer",
+        "expires_in":3600
+    }
+    ```
+
+---
+
+## Obtain the requesting party token
+
+The client acting on behalf of the requesting party has to obtain the requesting party token (RPT) with the obtained permission ticket and the claim token.
+
+Execute the following curl command to obtain the RPT.  
+
+-   Make sure to replace the `<CLIENT_ID>` and `<CLIENT_SECRET>` tags with the values you got after [configuring service provider for the client](#configure-service-provider-to-act-as-the-client).
+-   Replace `<PERMISSION_TICKET>` with the value you generated under the [obtaining a permission ticket](#obtain-a-permission-ticket) section.
+-   Make sure to replace the `<ID_TOKEN>` tag with the [OIDC id\_token](#obtain-the-oidc-id95token) you obtained.
+
+```
+curl --user <CLIENT_ID>:<CLIENT_SECRET> -k -d "grant_type=urn%3Aietf%3Aparams%3Aoauth%3Agrant-type%3Auma-ticket&ticket=<PERMISSION_TICKET>&claim_token=<ID_TOKEN>" -H "Content-Type: application/x-www-form-urlencoded" https://<IS_HOST>:<IS_PORT>/oauth2/token
+```
+
+You will get a response similar to the following:
+
+```
+{
+   "access_token":"p8dj48ff-heah-3632-b3dc-68aenm4c62e9",
+   "token_type":"Bearer",
+   "expires_in":3600
+}
+```
+
+---
+
+## Token introspection
+
+
+- Execute the following curl command to get the token introspection:  
+
+    -   Make sure to replace the `<PAT>` tag with the [access token you got in the previous section](#obtain-the-protection-api-access-token-pat).
+
+    -   Replace the `<RPT>` tag with the [request party token you obtained](#obtain-the-requesting-party-token).
+
+    ``` java
+    curl -v -k -H "Authorization: Bearer <PAT>" -H "Content-Type:application/x-www-form-urlencoded" -X POST --data "token=<RPT>" https://<IS_HOST>:<IS_PORT>/oauth2/introspect
+    ```
+
+-   You get a response similar to the following:
+
+    ```
+    {
+    "nbf":1553414959,
+    "permissions":[
+        {
+            "resource_scopes":[
+                "view"
+            ],
+            "resource_id":"08e69621-e418-4914-b85f-164e127c20de"
+        }
+    ],
+    "active":true,
+    "token_type":"Bearer",
+    "exp":1553418559,
+    "iat":1553414959,
+    "client_id":"JfTSiJ24gh8sYHTQVuOl5RoftkAa",
+    "username":"sam"
+    }
+    ```
+
+-   If the token introspection for the RPT is successful, the resource
+server can share the resource with the client.
+    -   In order to obtain UMA related information in the introspection endpoint, add the following configuration to the `deployment.toml` file in the `<IS_HOME>/repository/conf/` folder.  
+    -   This is disabled by default. The response shown above with additional UMA related details is what we get when the following configuration is enabled.
```

Comment:
I believe this point should be changed as given below and also moved to the last point.

```
This configuration is disabled by default. You will get the response shown above with additional UMA-related details when the configuration is enabled.
```